.github/workflows/make-release.yml

```diff
@@ -0,0 +1,107 @@
+name: 'Yakut application builder'
+
+# Builds Yakut release
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+# Ensures that only one workflow is running at a time
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  #
+  # Yakut test
+  #
+  yakut-test:
+    name: Test Yakut
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-2019-npcap]
```

Comment:
Is it possible to use Ubuntu-latest?

Reply:
Note, changing to `ubuntu-latest` caused `mypy --strict` to fail for an inline comment in main.py on line 291. I removed this inline comment.